js/views/userPageVw.js

```diff
@@ -303,10 +303,19 @@ UserPageVw = pageVw.extend({
             model.set('headerURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').header_hash + "&guid=" + self.pageID);
             model.set('avatarURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').avatar_hash + "&guid=" + self.pageID);
           }
+
           // Cache user avatar in localStorage
           var profile = model.toJSON().profile;
           window.localStorage.setItem("avatar_" + self.pageID, profile.avatar_hash);
 
+          if (response.profile.website) {
+            var website = profile.website;
+            var parse_url_regex = /^(?:([A-Za-z]+):)?(\/{0,3})([0-9.\-A-Za-z]+)(?::(\d+))?(?:\/([^?#]*))?(?:\?([^#]*))?(?:#(.*))?$/;
+            var parts = parse_url_regex.exec(website);
+            profile.short_website = parts[3];
+            model.set('profile', profile);
```

Comment:
Since the profile model may be used in other views, and we only need short_website in this view, it's better to pass it directly to the template.

Use

self.short_website = [value]

here instead.

Then do this in the render:

self.$el.html(loadedTemplate(
          __.extend(self.model.toJSON(), {
            backToTopTmpl: backToTopTmpl,
            short_website: self.short_website
          })
        ));

If you do that, change the template to use ob.short_website.